uwuno/JSON Documentation.txt

```diff
@@ -18,7 +18,17 @@ http://localhost:8080/rooms/<uid>
 {
     uid,
     roomName,
-    roomStatus: lobby // state of the game either: Lobby, Start, End
+    roomStatus: lobby // state of the game either: Lobby, Start, Restart (for Point Mode where there are multiple rounds)
+    gameMode       // normal or point
+}
+
+Game settings
+{
+  uid,
```

Comment:
I added it for PUT following the same pattern as what we had for PUT in playerController. But now I think we might want a different endpoint e.g. `rooms/uid/settings` instead of `rooms/uids` what do you think?

I will add it for POST that match the same endpoint.

Reply:
Actually. I think I will use the settings endpoint and add GET too.